tarofreighter/chain_porter.go

```diff
@@ -936,6 +937,8 @@ func (p *ChainPorter) stateStep(currentPkg sendPackage) (*sendPackage, error) {
 				},
 			},
 			TapscriptSibling: currentPkg.InputAsset.TapscriptSibling,
+			// TODO(bhandras): use clock.Clock instead.
+			TransferTime: time.Now(),
```

Comment:
Good move to also move this into the application layer so the DB doesn't explicitly need to worry about time itself.